CODEOWNERS

```diff
@@ -363,6 +363,7 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 /contrib/vcl/ @florincoras @KfreeZ
 /contrib/hyperscan/ @zhxie @soulxu
 /contrib/language/ @realtimetodie @realtimetodie
-/contrib/dlb/ @mattklein123 @daixiang0
+# TODO(phlax): move this extension
```

Comment:
nit: add tracking issue.

Reply:
https://github.com/envoyproxy/envoy/issues/29550

Reply:
nit: add the issue# to the comment